src/mesido/esdl/esdl_heat_model.py

```diff
@@ -1948,6 +1948,7 @@ def convert_electrolyzer(self, asset: Asset) -> Tuple[Type[Electrolyzer], MODIFI
         Optional ESDL fields:
         ---------------------
         - technicalLifetime
+        - powerFactor
```

Comment:
Update the CHANGELOG

Reply:
Updated. Also added the ESDL example creation to the changelog.